extensions/wikia/AdEngine3/src/templates/background-changer.js

```diff
@@ -0,0 +1,57 @@
+const SCSS_FILE_PATH = '/skins/oasis/css/core/breakpoints-background.scss';
+
+function configureClasses(options) {
+	if (options.skinImage !== '' && options.skinImageWidth > 0 && options.skinImageHeight > 0) {
+		document.body.classList.add('background-not-tiled');
+		document.body.classList.add('background-fixed');
+		document.body.classList.add('background-dynamic');
+
+		if (options.ten64) {
+			document.querySelector('.background-image-gradient').classList.add('no-gradients');
+		}
+	} else {
+		document.body.classList.remove('background-dynamic');
+		document.body.classList.remove('background-not-tiled');
+		document.body.classList.remove('background-fixed');
+	}
+}
+
+class BackgroundChanger {
```

Comment:
Inspired by https://github.com/Wikia/app/blob/61d6d1b394768cf9ea78a22488d40750ce7eec35/skins/oasis/js/BackgroundChanger.js :)

Reply:
Yes, I didn't want to hack and use original backgroundChanger attached to window. I beliebe we can remove BackgroundChanges.js once we finish with AE3. It seems that it's used only by AE2.